airframe-sql/src/test/scala/wvlet/airframe/sql/analyzer/TypeResolverTest.scala

```diff
@@ -294,7 +272,14 @@ class TypeResolverTest extends AirSpec {
 
     test("resolve aggregation key with union") {
       val p = analyze("select count(*), id from (select * from A union all select * from B) group by id")
-      p.asInstanceOf[Aggregate].groupingKeys(0).child shouldBe MultiColumn(List(ra1, rb1), Some("id"), None)
+      p.asInstanceOf[Aggregate].groupingKeys(0).child shouldBe MultiColumn(
+        List(
+          AllColumns(None, Some(Seq(ra1, ra2)), Some(NodeLocation(1, 34))),
+          AllColumns(None, Some(Seq(rb1, rb2)), Some(NodeLocation(1, 60)))
```

Comment:
`id` is currently resolved as `select * from A union all select * from B` but it should be resolved as `ResolvedAttribute` of the corresponding column.